### ISSUE TITLE
Send rest of lipid/metabolomic timeseries samples 

### ISSUE THREAD

**Issue by JillAshey** [urol-e5/timeseries_molecular#3]
Decided in 10/18 e5 meeting to send the rest of timeseries samples (96 samples total). 

Important links 
- [Samples to send](https://docs.google.com/spreadsheets/d/1iFsVfp1vix9IfNcqSajQfLgXXDg0CCPT/edit?gid=374085219#gid=374085219)
- Lipid / metabolites host separation [protocol](https://docs.google.com/document/d/1EiO1sB_tROCc0xWuQr3LXpvTtNoSSY8oLlTLwOQC1eo/edit)
- Jill's initial test sample [protocol and information](https://github.com/JillAshey/JillAshey_Putnam_Lab_Notebook/blob/master/_posts/2024-08-29-e5-Airbrushing-Metabolomics-Test-Samples.md)

To do 
- Find samples in -80C (need help from Zoe) 
- Order materials (~500 1.5 mL tubes, ~150 plastic pestles capable of fitting in 1.5mL tubes, MAYBE 2 liters of LCMS grade water - we have nuclease free water in the lab, not sure who is using it or if it is being saved for a specific purpose)
- Decide on tube labels

**Comment by AHuffmyer** [comment]
We can use the following protocol to prepare LC grade MilliQ water without the need to purchase LC grade water. This is how I generated LC grade MilliQ water for my metabolomic/lipidomic work in Australia. 

In order for water to be considered LC grade following MilliQ filtration, it must be in specially cleaned glassware. 

Any glass container used to prepare or store mobile phase (e.g. methanol, water) must be thoroughly cleaned before use to be LC clean. To clean laboratory glassware:

a. 	First, rinse it with organic solvent (e.g. methanol) and then water.
b. 	Next, rinse it with MilliQ water.
c. 	If more aggressive cleaning is required (for example, when the container’s history is unknown), use the following procedure: Sonicate with 10% formic or nitric acid, then water, then methanol or acetonitrile, then water. Repeat two more times. If you have clean glassware that is not heavily soiled, this step is not necessary. 
d. 	Wash and store glassware separately from other containers.

CAUTION: Do not wash glass bottles in detergent, with other glassware, or in washing facilities that may have detergent residue. Washing glassware in a common dishwashing facility can contaminate it with detergent residues, which may contain polyethyleneglycol (PEG) and other “sticky” substances. Vinyl-coated steel racks can be an additional source of contamination. Once glassware is cleaned, it can be used multiple times as long as there are not possible sources of contamination. 

Label the glassware "LC grade MilliQ" to use for our purposes. Follow the same procedure for glassware used to store any 10X PBS generated with this LC grade MilliQ water.

**Comment by AHuffmyer** [comment]
Note that nuclease free water is different from LC grade water. For LC grade water, we have to ensure that the water does not contain contaminants that can stick onto lipids and therefore influence our analyses or introduce contamination signals. 

@JillAshey please follow the procedure above to generate LC grade MilliQ water for our use rather than purchase.

**Comment by AHuffmyer** [comment]
Note that we are still waiting on metabolomics data, so we need to wait to ensure those data look good before giving the final green light. But because lipids looked good, chance is high that metabolites will also work.

**Comment by hputnam** [comment]
We have 10 bottles of molecular biology grade water (1L) stored on molecular bench shelves. You are welcome to use these if they work for you. https://geneseesci.com/shop-online/product-details/25-969/quality-biological-inc-351-029-131cs-molecular-biology-water-10-x-1000ml-10-bottles-unit

**Comment by sr320** [comment]
I recommend buying LcMs water if needed - we can purchase and ship to RI.

**Comment by hputnam** [comment]
@JillAshey what water did you use for the tests?  

I agree with @AHuffmyer "Note that we are still waiting on metabolomics data, so we need to wait to ensure those data look good before giving the final green light."

**Comment by JillAshey** [comment]
Agreed we should wait to move forward until after we get the metabolomics data back! 

I used dpec treated water that we had in 50 mL containers. This is the water we currently have in the lab: 

![IMG_7710](https://github.com/user-attachments/assets/3f411fd2-5e63-4c67-ba74-8401414ba355)

**Comment by AHuffmyer** [comment]
We will want to use LC-grade water specifically for the full data set. The molecular water was just fine for our tests, but for the full set we will want to use LC grade to ensure ions or other contaminants are not present that could interfere with ion detection.

**Comment by JillAshey** [comment]
sounds good @AHuffmyer!

**Comment by sr320** [comment]
is this complete?

**Comment by AHuffmyer** [comment]
Received at UW
